packaging/linux/linux_agent_builder.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -eux
+
+DISTRO="${1:-centos}"
+RELEASE="${2:-core}"
+CFY_VERSION="$3"
+
+AGT_NAME="$DISTRO-$RELEASE-agent_$CFY_VERSION"
+WORKDIR="$(pwd)"
+
+mkdir -p cloudify/env
+
+# This is going to create an agent .tar.gz with a python inside the agent's
+#   env/ folder, not a true virtualenv.
+
+# -- copy python 3.10 to "venv"
+curl https://cloudify-cicd.s3.amazonaws.com/python-build-packages/cfy-python3.10.tgz -o cfy-python3.10.tgz
```

Comment:
We could use "--retry 5" (or some other value) on this if we think there might be occasional connection issues with s3 (which is not an unreasonable assumption).

Reply:
It'd also be really nice if we looked at putting at least some hashes to check to avoid downloading a broken package and only finding out later, but I think that's probably for later.

Reply:
As noted on the channel, curl is really stupid about return codes: it'll happily return 0 on errors. It needs `-f` to actually abort on failures.
So I suggest this:
```suggestion
curl -f --retry 5 https://cloudify-cicd.s3.amazonaws.com/python-build-packages/cfy-python3.10.tgz -o cfy-python3.10.tgz
```